### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by OwenRay**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (mvdhoop) [inline index.html]: I think this could be in one meta tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    <meta name="viewport" content="width=device-width, user-scalable=no" />
```
Comment (mvdhoop) [inline index.html]: I think these could be in one meta tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    <meta property="og:title" content="TLNT - Changing the casting industry">
+    <meta property="og:url" content="http://tlnt.io">
+  </head>
+  <body>
+
+    <nav class="navbar navbar-expand-lg navbar-light">
+      <div class="container">
+        <a class="navbar-brand" href="#">
+          <div class="logoWrap">
+            <img src="assets/images/logo-header.svg" alt="Logo TLNT">
```
Comment (mvdhoop) [inline index.html]: All content locations should start with /

File: index.html [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+      </div>
+    </nav>
+
+    <header>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-5 col-sm-12">
+            <span class="made-by">The team of iCasting</span>
+            <h1>proudly introduces <span>The TLNT coin!</span></h1>
+            <p>Changing the casting industry by creating a thriving and bustling community where talents & clients are fully in control of their own personal profile data and financial transactions.</p>
```
Comment (mvdhoop) [inline index.html]: & sign should be changed to `&amp;`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+              <a data-toggle="modal" data-target="#videoModal" href="#">
+                <img class="video-preview__image" src="assets/images/video-grab.png" alt="Click here to open video">
+                <img class="video-preview__play-icon" src="assets/images/Play-icon.svg" alt="Click here to open video">
+              </a>
+            </div>
+          </div>
+          <div class="col-md-5 content content--right">
+            <div class="image-block"></div>
+            <div class="align-self-center">
+              <h2>iCasting & TLNT in <span>a nutshell</span></h2>
```
Comment (mvdhoop) [inline index.html]: & sign should be changed to `&amp;`

File: index.html [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+              <p>Our goal is to create a self-sustaining market, connecting talents with clients worldwide & without barriers, by  incorporating the blockchain technology.</p>
```
Comment (mvdhoop) [inline index.html]: & sign should be changed to `&amp;`

**Review by mvdhoop — CHANGES_REQUESTED** [review]
Couple of small changes

File: index.html (hunk 2 above)
Comment (DemienDrost) [reply index.html]: ☑️

File: index.html (hunk 3 above)
Comment (DemienDrost) [reply index.html]: ☑️

File: index.html (hunk 4 above)
Comment (DemienDrost) [reply index.html]: ☑️

File: index.html (hunk 5 above)
Comment (DemienDrost) [reply index.html]: ☑️

File: index.html (hunk 6 above)
Comment (DemienDrost) [reply index.html]: ☑️

**Review by mvdhoop — APPROVED** [review]

**Pull request merged**